modules/sms_email_reminder/cron_email_notification.php

```diff
@@ -1,59 +1,74 @@
 <?php
-
-/*
+/**
  * Purpose: to be run by cron every hour, look for appointments
  * in the pre-notification period and send an email reminder
  *
  * @package OpenEMR
  * @author Larry Lart
  * @copyright Copyright (c) 2008 Larry Lart
+ * @copyright Copyright (c) 2022 Luis A. Uriarte <luis.uriarte@gmail.com>
  * @link https://www.open-emr.org
  * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 // comment below exit if plan to use this script
-exit;
+//exit;
```

Comment:
better to either leave the exit or remove it completely (leaving it uncommented will just confuse people)

Reply:
Only from command line: cron or scheduled tasks

Reply:
then would place the following here (and remove the exit stuff):
```
if (php_sapi_name() !== 'cli') {
    exit;
}
```